loopd/run.go

```diff
@@ -168,10 +194,14 @@ func Run(rpcCfg RPCConfig) error {
 
 	lisCfg := newListenerCfg(&config, rpcCfg)
 
+	// Start listening for signal interrupts regardless of which command
+	// we are running. When our command tries to get a lnd connection, it
+	// blocks until lnd is synced. We listen for interrupts so that we can
+	// shutdown the daemon while waiting for sync to complete.
+	signal.Intercept()
```

Comment:
This currently has the effect that the view command cannot be canceled anymore because it does not react to the shutdown. But I don't think that matters hugely as it's a command that should normally terminate quickly.

Reply:
We can still terminate view if lnd isn't synced, which is what we need the interrupt for? 
Could also add intercept checks to the view function between the various calls, but I don' think that's required as you say.

Reply:
Yeah, you're right, without this you couldn't cleanly shut down the daemon on a view command if lnd wasn't synced. And view probably doesn't take very long anyway. It's not like the server side view command that has to interact with `bitcoind` and could potentially take a while.

Reply:
Yeah def, if it becomes more long lived we'll need to check for interrupts 👍